client/state/action-types.js

```diff
@@ -21,6 +23,8 @@ export const COMMENTS_REQUEST_SUCCESS = 'COMMENTS_REQUEST_SUCCESS';
 export const COMMENTS_REQUEST_FAILURE = 'COMMENTS_REQUEST_FAILURE';
 export const COMMENTS_UNLIKE = 'COMMENTS_UNLIKE';
 export const CURRENT_USER_ID_SET = 'CURRENT_USER_ID_SET';
+export const CUSTOMIZER_CLOSED = 'CUSTOMIZER_CLOSED';
```

Comment:
Oh, you're right about that. I personally don't work on the customizer, so I can't guarantee if/when they're migrated -- I just wanted to turn strings into constants. But I'm going to revert the relevant commit.